asciidoc/courses/aura-dashboards/modules/2-understand-your-data/lessons/1-describe-data-model/questions/1-structure.adoc

```diff
@@ -0,0 +1,27 @@
+[.question]
+= Structure of your data model
+
+Select the correct options that describe the components of your data model:
+
+* [x] Properties
+* [x] Relationships.
+* [ ] Instances.
+* [x] Nodes.
+
+[TIP,role=hint]
+.Hint
+====
+Think about the hierarchy you learned about in the lesson. Which components are part of the structure?
+====
+
+[TIP,role=solution]
+.Solution
+====
+The correct options that describe the structure components of your data model are:
+* **Properties**: These are key-value pairs that provide additional information about nodes and relationships.
+* **Relationships**: These define how nodes are connected to each other in the graph.
+* **Nodes**: These are the primary entities in the graph that represent data points.
```

Comment:
In many other places we stress how in our graph db, both nodes AND relationships are of equal importance and are stored accordingly, which is where the power of graph lies, ie no joins involved in multi-hop traversals.
That's why I wouldn't say that nodes are the _primary_ entities in the graph.